scripts/get_oppia_classes.py

```diff
@@ -0,0 +1,196 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""get_oppia_classes.py
+This script is used to get a dictionary with directive files as keys and
+oppia classes(belonging to them uniquely) as values.
+This can be used to shift all the classes which are used only in single
+files from oppia.css to respective files
+The scripts ask for the path to directory in which you want to run it.
+The script will search among all the files that lie below that directory
+in file structure i.e it will search in subdirs recursively also.
+
+====================================
+CUSTOMIZATION OPTIONS FOR DEVELOPERS
+====================================
+You can also use this script for similar use cases but different type of
+files by changing the variables html_pattern, directive_pattern and class
+_pattern respectively
+"""
+
+import os
+import re
+import bs4
+
+
+def return_all_files(dir_to_scan):
+    """
+    returns all files that lie below that directory in file structure
+
+    ARGS: directory name
+    RETURNS: all files in the path
+    """
+    file_result = []
```

Comment:
Perhaps better to call this `filepaths`

Btw, when you refer to "files" it implies the actual file. But what you are working with here is the references to these files, not the files themselves. So, could you please change 'file' to 'filepath' where appropriate?